### DISCUSSION TITLE
Resultados com valor abaixo do modificador da pericia

### DISCUSSION THREAD

**Discussion by Luukazer** [alyssapiresfernandescefet/openrpg#16] [Q&A] [answered]
Primeiramente gostaria de agradecer por todo esse trabalho, ficou incrivel, assim que conseguir uma graninha extra pretendo fazer uma doação por esse trabalho maravilhoso !!!

Mas minha dúvida é a seguinte, meus jogadores estão rolando as pericias e duas coisas acontecem, que não sei é bug ou se está projetado para funcionar assim. 
Primeiro os resultados deles são diferentes do que aparecem no OBS, por exemplo o jogador rola e tira um 25 porém no OBS aparece o resultado 20 (já verifiquei tanto no histórico e os valores que jogadores me dizem são verdadeiros). 
O outro "problema" que estou tendo é que quando meus jogadores rolam a pericia, mesmo que tenham um modificador nela de +5 por exemplo, a rolagem (as vezes) dar um valor menor do que do modificador base. Exemplo: O jogador rola a pericia de "atirar" e tira o resultado 1, mesmo tendo +5 na perícia. O resultado esperado seria 6, creio eu.

**Comment by lucaspiresfernandes** [comment, marked as answer]
Olá, @Luukazer, fico feliz que você gostou do aplicativo!

Existe um bug conhecido no Open RPG onde ao selecionar a opção "Rolar Novamente" na rolagem de dados, o modificador não é aplicado. Ou seja, apenas na primeira rolagem de dados que o modificador é aplicado.

Esse erro foi corrigido numa atualização que já lancei (5dc82c36f6d093f644fa6c1a708ad0167387600e), então atualize seu site pra aplicar essa correção.

**Reply by Luukazer** [reply]
MUITO OBRIGADO !!! vou atualizar aqui, você é demais
